site/.vitepress/config.ts

```diff
@@ -518,6 +518,6 @@ export default defineConfig({
         defer: "defer",
       },
     ],
-    ["link", { rel: "icon", href: "/public/favicon.ico" }],
+    ["link", { rel: "icon", href: "/favicon.ico" }],
```

Comment:
weird, why was this working in staging and not prod?

Reply:
it was working, but vitepress just gave a warning about it